feature-libs/cart/import-export/components/export-entries/export-order-entries.component.ts

```diff
@@ -15,6 +15,8 @@ import { ExportOrderEntriesToCsvService } from './export-order-entries-to-csv.se
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class ExportOrderEntriesComponent {
+  @HostBinding('class') styles = 'container';
```

Comment:
Is there a reason why you used hostbinding here instead of applying the styles directly?

Reply:
@bit172 
I don't think you can add a style class in the template for the host element ...
Do you mean like below in the component property?
![image](https://user-images.githubusercontent.com/11823988/157561514-8e8f21d2-33f9-4743-87b0-6f058ded253b.png)

Thanks

Reply:
@bit172 
I used the existing "container"  class (I think this class coming from bootstrap) because its parent's children containers use this class  (e.g.cx-order-confirmation-items) so it should have the same behavior in different browser sizes.